kenji_crosland/public/index.html

```diff
@@ -0,0 +1,21 @@
+<!DOCTYPE "html">
+<html>
+<head>
+  <title></title>
+</head>
+  <body>
+  <h1>Hello{{Name}}!</h1>
+    <p class="route-descriptions">
+      Here are the amazing things you can do with the app!
+      <ul class="routes">
+        <li class="routedescription">
+          Type in http://localhost:3000/greet/(anyname) and we will greet you by that name! Example: <a href="/greet/Cornelius">http://localhost:3000/greet/Cornelius</a>
```

Comment:
I would wrap this test in a <p> tag even tho it's inside an <li>, because it's enough text to do that.  This is one of those grey areas of html and keeping things clean... so take this comment with a grain of salt